bigchaindb/commands/bigchaindb.py

```diff
@@ -95,21 +95,35 @@ def run_configure(args):
 
 @configure_bigchaindb
```

Comment:
Do we need to set the `configure_bigchaindb` decorator here? Everything is already configured. Or am I missing something?

Reply:
No idea. It's not part of my PR